packages/x-charts/src/themeAugmentation/components.d.ts

```diff
@@ -18,6 +18,10 @@ export interface ChartsComponents {
   MuiChartsClipPath?: {
     defaultProps?: ComponentsProps['MuiChartsClipPath'];
   };
+  MuiChartsGrid?: {
+    defaultProps?: ComponentsProps['MuiChartsGrid'];
```

Comment:
I haven't tested it, but by the looks of the code—this won't work. 🤔 
Have you tested it?
It seems to be missing a call to `useThemeProps` to make it work. 🤷

Reply:
There is only 4 instances of `useThemeProps` in x-charts. 🤔